### PR TITLE
added a method for turning the chat type into a moderated one

### PR DESCRIPTION
**Pull request by leandr92**
Also may need to change chat creation method for add the feature for pass creation params which they may be more then "members" and "admins"

### CODE REVIEW

**Comment by leandr92** [comment]
PR changed by new commit

File: skpy/chat.py [hunk 1]
```diff
@@ -340,7 +340,7 @@ class SkypeGroupChat(SkypeChat):
             still appears in the recent list despite being left or deleted.
     """
 
-    attrs = SkypeChat.attrs + ("topic", "creatorId", "userIds", "adminIds", "open", "history", "picture", "active")
+    attrs = SkypeChat.attrs + ("topic", "creatorId", "userIds", "adminIds", "open", "history", "picture", "active", "moderatedthread")
```
Comment (Terrance) [inline skpy/chat.py:343]: This should be `moderated` for consistency with the method to change it.  It also needs adding to the docstring.

File: skpy/chat.py [hunk 2]
```diff
@@ -360,7 +360,8 @@ def rawToFields(cls, raw={}, active=False):
                        "open": props.get("joiningenabled", "") == "true",
                        "history": props.get("historydisclosed", "") == "true",
                        "picture": props.get("picture", "")[4:] or None,
-                       "active": active})
+                       "active": active,
+                       "moderatedthread": props.get("moderatedthread")})
```
Comment (Terrance) [inline skpy/chat.py:364]: This isn't a boolean -- as it stands, the value will be `"true"` or `"false"`, or `None` on older chats that are missing this setting.  It should be checked for equality to `"true"`, as is done with the `open` and `history` attributes.

File: skpy/chat.py [hunk 3]
```diff
@@ -492,27 +504,29 @@ def chat(self, id):
...
         return self.merge(SkypeChat.fromRaw(self.skype, json))
 
-    def create(self, members=(), admins=()):
+    def create(self, members=(), admins=(), moderated=False):
         """
         Create a new group chat with the given users.
 
         The current user is automatically added to the conversation as an admin.  Any other admin identifiers must also
-        be present in the member list.
+        be present in the member list. You can also make the chat moderated by passing moderated value
```
Comment (Terrance) [inline skpy/chat.py:512]: This should just be stated in the argument description below, which currently doesn't describe its use.

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+manual_test.py
+__pycache__/
+*.tokens-app
```
Comment (Terrance) [inline .gitignore:3]: The first and last of these are presumably your own files, and aren't relevant to the library itself.  `__pycache__` is valid, but better handled system-wide with a personal gitignore file on your own machine.

**Review by Terrance — COMMENTED** [review]
Not quite there, but I can probably make the remaining changes.

**Review by Terrance — APPROVED** [review]

**Pull request merged**